From 66c3abddd7eb2915d90687b7aa9e9ad5ed187a6d Mon Sep 17 00:00:00 2001
Message-ID: <66c3abddd7eb2915d90687b7aa9e9ad5ed187a6d.1703031694.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1703031693.git.sharlatanus@gmail.com>
References: <cover.1703031693.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 16 Dec 2023 04:18:49 +0000
Subject: [PATCH 16/32] gnu: python-photutils: Update to 1.10.0.

* gnu/packages/astronomy.scm (python-photutils): Update to 1.10.0.
[propagated-inputs]: Add python-bottleneck, python-gwcs, python-matplotlib,
python-rasterio, python-scikit-image, python-scikit-learn,
python-scipy, python-shapely, and python-tqdm.
[native-inputs]: Add python-semantic-version.

Change-Id: Ic2a879b653b56b84fff826b85108088db5cbf888
---
 gnu/packages/astronomy.scm | 17 ++++++++++++++---
 1 file changed, 14 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 2017b791c7..1b707ffc56 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2066,13 +2066,13 @@ (define-public python-mpl-animators
 (define-public python-photutils
   (package
     (name "python-photutils")
-    (version "1.9.0")
+    (version "1.10.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "photutils" version))
        (sha256
-        (base32 "0gjvrmcb10lyqx3h9521zqskqngd5xcp3l9h2b02mk1p7q54mzdy"))))
+        (base32 "148zhdxhlcgj6dxyzaz78bzxw1q44qa9q9sfdqbfbla0s0w2jaay"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -2091,11 +2091,22 @@ (define-public python-photutils
               (make-file-writable "photutils/_compiler.c")
               (invoke "python" "setup.py" "build_ext" "--inplace"))))))
     (propagated-inputs
-     (list python-astropy python-numpy))
+     (list python-astropy
+           python-bottleneck
+           python-gwcs
+           python-matplotlib
+           python-numpy
+           python-rasterio
+           python-scikit-image
+           python-scikit-learn
+           python-scipy
+           python-shapely
+           python-tqdm))
     (native-inputs
      (list python-cython
            python-extension-helpers
            python-pytest-astropy
+           python-semantic-version
            python-setuptools-scm))
     (home-page "https://github.com/astropy/photutils")
     (synopsis "Source detection and photometry")
-- 
2.41.0

